src/components/FocusTrap/FocusTrapForScreen/index.web.tsx

```diff
@@ -42,7 +42,11 @@ function FocusTrapForScreen({children, focusTrapSettings}: FocusTrapProps) {
             containerElements={focusTrapSettings?.containerElements?.length ? focusTrapSettings.containerElements : undefined}
             focusTrapOptions={{
                 onActivate: () => {
-                    (document?.activeElement as HTMLElement)?.blur();
+                    const activeElement = document?.activeElement as HTMLElement;
+                    if (activeElement?.nodeName === 'INPUT' || activeElement?.nodeName === 'TEXTAREA') {
```

Comment:
Can we create a const for both of these strings and update any usages in the app? I see quite a few.